apps/crn-frontend/package.json

```diff
@@ -6,11 +6,10 @@
     "node": "18.x"
   },
   "scripts": {
-    "start": "cross-env TSC_COMPILE_ON_ERROR=true react-scripts start",
-    "build": "node ./scripts/build-unsupported-browser-page.js && DISABLE_ESLINT_PLUGIN=true react-scripts build",
+    "start": "cross-env TSC_COMPILE_ON_ERROR=true vite start",
```

Comment:
I believe we should have

```suggestion
    "start": "cross-env TSC_COMPILE_ON_ERROR=true vite",
```

according to this https://www.freecodecamp.org/news/how-to-migrate-from-create-react-app-to-vite/

<img width="820" alt="Screenshot 2023-12-05 at 08 03 58" src="https://github.com/yldio/asap-hub/assets/16595804/ca80a44d-1b10-4198-a087-935bfb56998d">